projects/1.20/assets/map-atlases-forge/map_atlases/lang/zh_cn.json

```
@@ -0,0 +1,35 @@
+{
+  "item.map_atlases.atlas": "地图册",
+  "item.map_atlases.end_atlas": "末地地图册",
+  "item.map_atlases.nether_atlas": "下界地图册",
+  "item.map_atlases.atlas.tooltip_maps": "地图：%d幅",
+  "item.map_atlases.atlas.tooltip_empty": "空地图：%d幅",
+  "item.map_atlases.atlas.tooltip_scale": "缩放等级：1:%s",
+  "item.map_atlases.atlas.tooltip_slice": "切面：%s",
+  "item.map_atlases.atlas.tooltip_type": "类型：%s",
+  "item.map_atlases.atlas.tooltip_slice_default": "切面：顶部",
+  "item.map_atlases.atlas.tooltip_err": "无效",
+  "item.map_atlases.atlas.tooltip_full": "已满！",
+  "item.map_atlases.atlas.tooltip_locked": "已锁定",
+  "key.map_atlases.open_minimap": "打开小地图GUI",
+  "key.map_atlases.zoom_in_minimap": "放大小地图",
+  "key.map_atlases.zoom_out_minimap": "缩小小地图",
+  "key.map_atlases.place_pin": "放置标记",
+  "item.map_atlases.dummy_filled_map": "地图",
+  "item.map_atlases.dummy_filled_map.dummy": "地图占位符",
+  "item.map_atlases.dummy_filled_map.desc": "接受任意已绘制的地图",
+  "category.map_atlases.minimap": "地图册（Map Atlases）按键设置",
```

习惯性一般是直接写{中文译名}{按键设置}

---

其实是为了防止中文翻译没有流通或翻得不好留的保险
```suggestion
  "category.map_atlases.minimap": "地图册按键设置",
```
可直接按这个改（）